centos_install_requirements.sh

```diff
@@ -43,6 +43,8 @@ sudo yum -y install \
   wget
 
 if [[ $DISTRO == "centos7" ]]; then
+  # Install pip since it is used by the k8s Ansible module
+  sudo yum install -y python-pip
```

Comment:
let's use dnf since it's available

Reply:
Ok, but it's centos7 and there are yum commands all over the script.